addon/templates/components/uni-mobile-number.hbs

```diff
@@ -1,12 +1,15 @@
 {{uni-select
+  class="uni-mobile-number__select-wrapper"
   placeholder=selectPlaceholder
   selected=language
   options=options
+  useAlias=true
```

Comment:
Yes, but I think it's a better practice to allow someone to be able to change this field instead of forcing it